File: C/add.c
#include<stdio.h>

int add(int number1, int number2);

void main(){
	int number1, number2;
	printf("Enter the first number: ");
	scanf("%d", &number1);
	printf("Enter the second number: ");
	scanf("%d", &number2);
	int sum;
	sum = add(number1, number2);
	printf("The sum of %d and %d is %d.", number1, number2, sum);
}

int add(int number1, int number2){
	int sum;
	sum = number1 + number2;
	return sum;
}

File: go/add.go
package main

import "fmt"

func add(number1, number2 int) int {
	return number1 + number2
}

func main() {
	var number1 int
	var number2 int
	var sum int
	fmt.Print("Enter the first number: ")
	fmt.Scan(&number1)
	fmt.Print("Enter the second number: ")
	fmt.Scan(&number2)
	sum = add(number1, number2)
	fmt.Printf("Sum of %d and %d is %d.", number1, number2, sum)
}
